WalletWasabi/WebClients/Mempool/MemPoolSpaceFeeProvider.cs

```diff
@@ -0,0 +1,90 @@
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using WalletWasabi.Bases;
+using WalletWasabi.Blockchain.Analysis.FeesEstimation;
+using WalletWasabi.Tor.Http;
+using WalletWasabi.Tor.Http.Extensions;
+using WalletWasabi.Tor.Socks5.Pool.Circuits;
+using WalletWasabi.WebClients.Wasabi;
+
+namespace WalletWasabi.WebClients.Mempool;
+
+public class MemPoolSpaceFeeProvider : PeriodicRunner, IThirdPartyFeeProvider
+{
+	public MemPoolSpaceFeeProvider(TimeSpan period, WasabiHttpClientFactory httpClientFactory) : base(period)
+	{
+		HttpClient = httpClientFactory.NewHttpClient(() => new Uri("https://mempool.space"), Mode.DefaultCircuit);
```

Comment:
Base URL could be made configurable. There are other instances of mempool.space software too, it's FOSS, anyone can run their own.

Reply:
First this should be in, I consider that a separate feature.

Reply:
https://github.com/zkSNACKs/WalletWasabi/issues/13023 :)